packages/office-addin-debugging/src/start.ts

```diff
@@ -293,9 +293,6 @@ export async function startDebugging(manifestPath: string, options: StartDebuggi
         // enable debugging
         if (isDesktopAppType && isWindowsPlatform) {
             await devSettings.enableDebugging(manifestInfo.id, enableDebugging, debuggingMethod, openDevTools);
-            if (enableDebugging) {
```

Comment:
Sorry if this has been unclear but we want this console.log message. Just remove the "Debug method: " part at the end.